Server.cs

```diff
@@ -17,7 +17,7 @@ public class Server : Node
 
   AMQPserver MessageInterface;
 
-  [Export]
+  //[Export]
```

Comment:
Isn't there a different `@export` syntax in GD4? I don't think these should just be commented out. The exports were used during inspection in the editor to see what's in these various dicts and etc.

Reply:
From what I can tell the @export syntax is gdscript while the C# syntax remains the same. However, and importantly, [only certain collection type's can be exported](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_exports.html#exporting-collections). So will need to find an appropriate type if possible to retain the existing functionality.